cmd/statusd/main.go

```diff
@@ -68,7 +68,17 @@ var (
 		params.FleetProd,
 		fmt.Sprintf(
 			"Select fleet: %s (default %s)",
-			[]string{params.FleetProd, params.FleetStaging, params.FleetTest, params.FleetWakuV2Prod, params.FleetWakuV2Test, params.FleetStatusTest, params.FleetStatusProd}, params.FleetProd,
+			[]string{
+				params.FleetProd,
+				params.FleetStaging,
+				params.FleetTest,
+				params.FleetWakuV2Prod,
+				params.FleetWakuV2Test,
+				params.FleetStatusTest,
+				params.FleetStatusProd,
+				params.FleetShardsTest,
+				params.FleetShardsStaging,
```

Comment:
Maybe we should remove all old fleets that are no longer part of static sharding?

Reply:
Not sure, looking at fleets.status.im, those fleets are still active.
I think if we remove these fleets from this part of code, we'd likely need to make sure those fleets are gone too, otherwise a deployment of status-go to those fleets will likely fail.
cc: @jakubgs to confirm if this is correct

Reply:
The only fleets that should be removed are `wakuv2.*` fleets.

Reply:
Since the desktop and mobile apps are using only clusterID 16, I was wondering if there is a need for all the other Waku fleets that no longer use clusterID.